### PR TITLE
Fix issues introduced by the PDF manual merge

### LINKED ISSUE #8529: Formulas not displayed correctly in Sphinx
#### Description of the problem

Formulas don't render correctly any more.  IIRC they worked fine a week or two ago.  For example, in my locally built version of the doc in ..../coq/doc/sphinx/_build/html/language/cic.html#typing-rules:

image

@cpitclaudel do you know anything about this?

### PR DESCRIPTION
**Pull request by cpitclaudel**
**Kind:** documentation / bug fix

Fixes / closes #8529.  Also addresses two issues introduced with the PDF manual: the title of HTML pages had changed to 'The Coq Reference Manual vNN' instead of 'Coq vNN', and the 'credits.html' page had been renamed to 'credits-wrapper.html'.

### CODE REVIEW

**Comment by Zimmi48** [comment]
Thanks! Can you also fix the location of the first footnote of the introduction and the make targets? In particular, I need `install-doc-sphinx` to work again to be able to see the output of this PR as a GitLab CI artifact.

**Comment by Zimmi48** [comment]
@cpitclaudel I've pushed two commits on top of your (rebased) branch fixing the Makefile targets and the title of the Introduction chapter.

**Review by Zimmi48 — APPROVED** [review]
Approving the changes made by @cpitclaudel.

**Comment by Zimmi48** [comment]
Can I ask @gares @ejgallego or @SkySkimmer to review the changes I made in my own commit on `Makefile.doc`?

File: Makefile.doc [hunk 1]
```diff
@@ -199,7 +202,7 @@ install-doc-printable:
 	$(INSTALLLIB) doc/stdlib/Library.pdf $(FULLDOCDIR)/pdf
 	$(INSTALLLIB) doc/stdlib/Library.ps $(FULLDOCDIR)/ps
 
-install-doc-sphinx-html:
+install-doc-sphinx:
```
Comment (SkySkimmer) [inline Makefile.doc]: This one didn't get renamed to refman

Comment (Zimmi48) [reply Makefile.doc]: Indeed. I'm not happy with the current location where the manual is installed (see #8524) so until this is fixed, I prefer not to change the name of this target. (This is not urgent and I need to discuss this with @cpitclaudel before.)

Comment (ejgallego) [reply Makefile.doc]: @Zimmi48 I am not sure I am much competent on the make changes.

**Review by ejgallego — APPROVED** [review]
CI changes are trivial.

**Comment by Zimmi48** [comment]
@SkySkimmer Does this otherwise look good to you? I'll merge this in half an hour because it's the last thing blocking the 8.8.2 release.

**Comment by Zimmi48** [comment]
BTW, here it is rendered: https://coq.gitlab.io/-/coq/-/jobs/101278834/artifacts/_install_ci/share/doc/coq/sphinx/html/index.html and https://gitlab.com/coq/coq/-/jobs/101278834/artifacts/raw/_install_ci/share/doc/coq/sphinx/latex/CoqRefMan.pdf?inline=false

**Pull request merged**